baseplate/lib/prometheus_metrics.py

```diff
@@ -438,3 +438,59 @@ def get_requests_total_metric(cls) -> Counter:
     @classmethod
     def get_active_requests_metric(cls) -> Gauge:
         return cls.active_requests
+
+
+class PrometheusSQLAlchemyClientMetrics:
+    prefix = "database_client"
```

Comment:
There are lots of different kinds of databases, please name this more specifically, both the class and the prefix. 

```suggestion
class PrometheusSQLClientMetrics:
    prefix = "sql_client"
```

Reply:
In our current metrics, there are tags for the client name. This tag is populated using the same name used to attach the client to the baseplate context. This allows us to have metrics for multiple postgres databases in the same service. Will this continue to be possible?

Reply:
Good point, adding a label for the endpoint used is definitely needed.

Reply:
Would also add that the db host and db itself might be a good label to have as well.

The point I was trying to make is that this metrics class is more for SQL than it is for something like a redis to Cassandra DB.